uploaddocs.sh

```diff
@@ -66,12 +66,14 @@ do
     echo "Final upload stage (googleapis.dev)"
     python -m docuploader upload . --credentials $SERVICE_ACCOUNT_JSON --staging-bucket $GOOGLEAPIS_DEV_STAGING_BUCKET
 
-    cd ../devsite/api
+    # For DevSite, we just upload the yaml in the obj/api directory
+    cd ../obj/api
     echo "Generating metadata (DevSite)"
     python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
```

Comment:
```suggestion
    # FIXME: WE DON'T CURRENTLY KNOW API_SHORTNAME
    api_shortname=todo
    python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet --serving-path /dotnet/docs/reference/$api_shortname/$pkg/latest
```

Reply:
Ah, my mistake - I'd thought it was in the upload stage for some reason.